Makefile

```diff
@@ -279,6 +279,18 @@ test/k3s-%: dist/img.tar bin/k3s dist-%
 	sudo bin/k3s kubectl wait deployment wasi-demo --for condition=Available=True --timeout=5s
 	sudo bin/k3s kubectl get pods -o wide
 
+.PHONY: test/k3s-oci-%
+test/k3s-oci-%: dist/img-oci.tar bin/k3s dist-%
+	sudo bash -c -- 'while ! timeout 40 test/k3s/bootstrap.sh "$*" dist/img-oci.tar; do $(MAKE) bin/k3s/clean bin/k3s; done'
+	sudo bin/k3s kubectl get pods --all-namespaces
+	sudo bin/k3s kubectl apply -f test/k8s/deploy.oci.yaml
+	sudo bin/k3s kubectl get pods --all-namespaces
+	sudo bin/k3s kubectl wait deployment wasi-demo --for condition=Available=True --timeout=120s
+	# verify that we are still running after some time	
+	sleep 5s
+	sudo bin/k3s kubectl wait deployment wasi-demo --for condition=Available=True --timeout=5s
+	sudo bin/k3s kubectl get pods -o wide
```

Comment:
Could you please also add commands to verify that the pods can be deleted successfully?

Reply:
done